packages/app/source/server/projections/open-appeals-projection.js

```diff
@@ -0,0 +1,36 @@
+Space.eventSourcing.Projection.extend(Donations, 'OpenAppealsProjection', {
```

Comment:
yeah, probably we should really do that – we are wasting a lot of time splitting up files / creating extra classes / mappings for future concepts that are not yet there. All of that is premature optimization and just gives the impression that "you need to do a lot of boilerplate" when working with Space.

Especially the Meteor community is really sensitive to these kinds of things. That is because many come to Meteor exactly for that reason: it stays simple and compact.

Yeah … let's merge all collections / apis / projections (everything that is just boilerplate atm) into single files.